client/deployment_test.go

```diff
@@ -205,14 +205,8 @@ func TestLogs(t *testing.T) {
 		assert.Equal(t, "/logs", endpoint)
 
 		// Check body
-		body, err := ioutil.ReadAll(req.Body)
-		assert.Nil(t, err)
 		defer req.Body.Close()
-		var upReq common.DaemonRequest
-		err = json.Unmarshal(body, &upReq)
-		assert.Nil(t, err)
-		assert.Equal(t, "docker-compose", upReq.Container)
-		assert.Equal(t, true, upReq.Stream)
+		assert.Equal(t, "Container=docker-compose&Stream=true", req.URL.RawQuery)
```

Comment:
Since the daemonside handler uses this syntax:
```go
container := q.Get("Container")
```
could we use that here in the test as well, just to ensure consistency?

I also wonder if the argument names (`container` , `stream`) might be better to have as exported variables in package `common` alongside the Request structs?

Reply:
Good point! Yeah I agree that it would make sense to put them in a common folder, since they're used across different files.